### PR TITLE
UI/ux overhaul

### PR DESCRIPTION
**Pull request by BAFiogbe**
-- Needs data from Meal Plans and Shopping List

### CODE REVIEW

File: app/src/main/res/layout/activity_recipe_storage.xml [hunk 1]
```diff
@@ -10,12 +10,11 @@
         android:id="@+id/recipe_list"
         android:layout_width="383dp"
         android:layout_height="535dp"
-        android:layout_marginTop="70dp"
         app:layout_constraintBottom_toTopOf="@+id/return_text"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.444"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="@+id/SortType" />
+        app:layout_constraintTop_toBottomOf="@+id/recipe_filter" />
```
Comment (kstark1) [inline app/src/main/res/layout/activity_recipe_storage.xml:20]: add a back arrow button similar to the ingredient storage view

Comment (kstark1) [inline app/src/main/res/layout/activity_recipe_storage.xml:20]: add constraints to the text. They are all currently overlapping

File: app/src/main/res/layout/activity_login_screen.xml [hunk 2]
```diff
@@ -4,26 +4,41 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background = "@drawable/background__3_"
     tools:context=".userlogin.LoginActivity">
 
+
```
Comment (kstark1) [inline app/src/main/res/layout/activity_login_screen.xml:10]: make this a scroll view, currently cannot reach the register button

**Review by kstark1 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/happymeals/userlogin/LoginActivity.java [hunk 3]
```diff
@@ -43,7 +43,8 @@ public class LoginActivity extends AppCompatActivity implements InputStringFragm
 
     private Context context;
 
-    private Button login, register;
+    private Button login;
+    private TextView register;
```
Comment (jeastgaard) [inline app/src/main/java/com/example/happymeals/userlogin/LoginActivity.java:47]: What were these changes for? I see no other changes to the file. Why is Register a TextrView now, should still be button?

File: app/src/main/res/layout/activity_recipe_storage.xml (hunk 1 above)
Comment (BAFiogbe) [reply app/src/main/res/layout/activity_recipe_storage.xml:20]: I didn't edit or make this, and I also cannot find in the xml where the back arrow button is

**Review by jeastgaard — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/happymeals/userlogin/LoginActivity.java (hunk 3 above)
Comment (BAFiogbe) [reply app/src/main/java/com/example/happymeals/userlogin/LoginActivity.java:47]: On the login screen the register isn't a button anymore and is now clickable text

**Review by jeastgaard — APPROVED** [review]

**Review by kstark1 — APPROVED** [review]

**Pull request merged**